### PR TITLE
Add e2e tests for java auto-instrumentation

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <p.loffay@gmail.com>

### CODE REVIEW

**Comment by pavolloffay** [comment]
Please do not merge yet!

File: tests/e2e/instrumentation-java/01-assert.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    sidecar.opentelemetry.io/inject: "true"
+  labels:
+    app: my-pod-with-sidecar
+spec:
```
Comment (pavolloffay) [inline tests/e2e/instrumentation-java/01-assert.yaml:8]: I need to add more asserts here

**Comment by jpkrohling** [comment]
>Please do not merge yet!

You can also mark the PR as draft to prevent it from being merged :-)

**Comment by pavolloffay** [comment]
This is ready for review

**Comment by jpkrohling** [comment]
@VineethReddy02, could you please review this one?

**Review by VineethReddy02 — APPROVED** [review]
LGTM!

**Pull request merged**